File: src/main/java/tech/zolhungaj/amqapi/servercommands/objects/AnimeListSiteShowIds.java
package tech.zolhungaj.amqapi.servercommands.objects;

import com.squareup.moshi.Json;

import java.util.Optional;

public record AnimeListSiteShowIds(
        @Json(name = "kitsuId")
        Optional<Integer> kitsuId,
        @Json(name = "annId")
        Optional<Integer> animeNewsNetworkId,
        @Json(name = "malId") Optional<Integer> myAnimeListId,
        @Json(name = "aniListId") Optional<Integer> aniListId)
{}

File: src/main/java/tech/zolhungaj/amqapi/servercommands/objects/AnimeListSiteShowIds.kt
package tech.zolhungaj.amqapi.servercommands.objects

import com.squareup.moshi.Json

@JvmRecord
data class AnimeListSiteShowIds(
    @Json(name = "kitsuId") val kitsuId: Int?,
    @Json(name = "annId") val animeNewsNetworkId: Int?,
    @Json(name = "malId") val myAnimeListId: Int?,
    @Json(name = "aniListId") val aniListId: Int?
) 